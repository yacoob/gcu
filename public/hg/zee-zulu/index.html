<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <!--[if lt IE 9]>
  <script src='https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js'></script>
  <script src='https://oss.maxcdn.com/respond/1.4.2/respond.min.js'></script>
  <![endif]-->
  <link href='/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='//syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151115_002812.jpg' rel='image_src'/>
  <meta property='og:image' content='https://syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151115_002812.jpg'/>
  <meta name='description' content='tactical grace presents: HG 1/144 AMS-129M Zee Zulu'/>
  <link href='http://gcu.tactical-grace.net/hg/zee-zulu/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>HG 1/144 AMS-129M Zee Zulu | tactical grace</title>
  <meta content='HG 1/144 AMS-129M Zee Zulu | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/hg/zee-zulu/'>HG 1/144 AMS-129M Zee Zulu</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a href='/hg/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a href='/hg/zaku-i-sniper/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2015-11-21'>2015-11-21</h2>
  </div>
  <div class='row voffset-small'>
  <div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151023_210923.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-11-21/s400-p/IMG_20151023_210923.jpg' title='I can&#39;t believe it&#39;s still an HG!' alt='I can&#39;t believe it&#39;s still an HG!' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151023_233642.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-11-21/s400-p/IMG_20151023_233642.jpg' title='4k grit nail buffing stick works wonders.' alt='4k grit nail buffing stick works wonders.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151106_230354.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-11-21/s400-p/IMG_20151106_230354.jpg' title='So many extra pieces, why not plastic caps here instead of stickers, eh Bandai?' alt='So many extra pieces, why not plastic caps here instead of stickers, eh Bandai?' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151106_232243.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-11-21/s400-p/IMG_20151106_232243.jpg' title='This level of colour separation is uncommon in HG line.' alt='This level of colour separation is uncommon in HG line.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151115_002709.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-11-21/s400-p/IMG_20151115_002709.jpg' title='He&#39;s reaching for a knife. Or suffering some back pains.' alt='He&#39;s reaching for a knife. Or suffering some back pains.' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151115_002745.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-11-21/s400-p/IMG_20151115_002745.jpg' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='//syn.tactical-grace.net/c/2015-11-21/s1920/IMG_20151115_002812.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-11-21/s400-p/IMG_20151115_002812.jpg' title='*snikt*' alt='*snikt*' />
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='voffset-bottom-small'>
    <a class='gallery' rel='group' href='https://youtu.be/_4nB5vC1epE'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/c/2015-11-21/s400-p/zee-zulu-videothumb.PNG' />
    </a>
  </div>
</div>
  </div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='http://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link href='/css/jquery.fancybox.css' rel='stylesheet'>
<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js'></script>
<script src='/js/jquery.fancybox.min.js'></script>
<script src='/js/jquery.fancybox-media.js'></script>
<script src='/js/screenfull.min.js'></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
$(document).ready(gcu.postPageHandler);
</script>
</body></html>