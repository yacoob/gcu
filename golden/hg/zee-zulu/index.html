<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <link href='https://cdn.jsdelivr.net/npm/bootstrap3@3/dist/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='/photos/full/gcu-00701.jpg' rel='image_src'/>
  <meta property='og:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00701.jpg'/>
  <meta name='description' content='tactical grace presents: Zee Zulu'/>
  <link href='https://gcu.tactical-grace.net/hg/zee-zulu/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>HG: Zee Zulu | tactical grace</title>
  <meta content='HG: Zee Zulu | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/hg/zee-zulu/'>HG: Zee Zulu</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a data-toggle='tooltip' title='main page' href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a data-toggle='tooltip' title='contact the author' href='https://goo.gl/forms/iJEsfLjBy1FE86aQ2'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-envelope'></span></button></a>
    <a data-toggle='tooltip' title='grade page' href='/hg/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a data-toggle='tooltip' title='previous kit' href='/hg/zaku-tri-star/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2015-11-21'>2015-11-21</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00695.jpg' data-sub-html='I can&#39;t believe it&#39;s still an HG!'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00695.jpg' alt='I can&#39;t believe it&#39;s still an HG!' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00696.jpg' data-sub-html='4k grit nail buffing stick works wonders.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00696.jpg' alt='4k grit nail buffing stick works wonders.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00697.jpg' data-sub-html='So many extra pieces, why not plastic caps here instead of stickers, eh Bandai?'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00697.jpg' alt='So many extra pieces, why not plastic caps here instead of stickers, eh Bandai?' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00698.jpg' data-sub-html='This level of colour separation is uncommon in HG line.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00698.jpg' alt='This level of colour separation is uncommon in HG line.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00699.jpg' data-sub-html='He&#39;s reaching for a knife. Or suffering some back pains.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00699.jpg' alt='He&#39;s reaching for a knife. Or suffering some back pains.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00700.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00700.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00701.jpg' data-sub-html='*snikt*'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00701.jpg' alt='*snikt*' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='https://youtu.be/_4nB5vC1epE'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00702.jpg' />
      <span class='glyphicon glyphicon-play-circle gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='https://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/css/lightgallery.min.css" />
<script src="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/js/lightgallery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-thumbnail.js@1/dist/lg-thumbnail.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-zoom.js@1/dist/lg-zoom.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-video.js@1/dist/lg-video.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-fullscreen.js@1/dist/lg-fullscreen.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-autoplay.js@1/dist/lg-autoplay.min.js"></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
document.addEventListener('DOMContentLoaded', gcu.postPageHandler);
</script>
</body></html>